extensions/2.0/Khronos/KHR_avatar_mesh_annotation/README.md

```diff
@@ -0,0 +1,116 @@
+# KHR_avatar_mesh_annotation
+
+## Contributors
+
+- Ken Jakubzak, Meta
+- Hideaki Eguchi / VirtualCast, Inc.
+- K. S. Ernest (iFire) Lee, Individual Contributor / https://github.com/fire
+- Shinnosuke Iwaki / VirtualCast, Inc.
+- 0b5vr / pixiv Inc.
+- Leonard Daly, Individual Contributor
+- Nick Burkard, Meta
+
+## Status
+
+**Draft** – This extension is not yet ratified by the Khronos Group and is subject to change.
+
+## Dependencies
+
+Written against the glTF 2.0 specification.
+Dependent on: `KHR_avatar`  
```

Comment:
I have the same question for other extensions as well. What exactly does “Dependent on: `KHR_avatar`” mean here? This extension does not require data from KHR\_avatar, does it? Does it mean we should ignore this extension if KHR\_avatar is not found in the file? But the goal was to allow picking up only the required extensions from the suite, wasn't it? We might need an explanation of what “Dependent” specifically means here.